File: RockPaperScissors/RockPaperScissors.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    // Intro
    System.out.println("Lets play a game:");
    System.out.println(" ---- Rock Paper Scissors ----");
    System.out.println("");

    System.out.println("Paper -> Rock");
    System.out.println("Scissors -> Paper");
    System.out.println("Rock -> Scissors");
    System.out.println("");

    // Player selection
    System.out.print("You: ");
    String player = scanner.nextLine();
    
    if (player.equals("Rock") || player.equals("Paper") || player.equals("Scissors")) {
      // Randomize CPU
      int rdn = (int) Math.round(Math.random() * 3);
      String cpu = "";

      if (rdn == 0)
        cpu = "Rock";
      else if (rdn == 1)
        cpu = "Paper";
      else
        cpu = "Scissors";

      System.out.println("CPU: " + cpu);

      // Check Winner
      System.out.println("");
      if (player.equals(cpu)) {
        System.out.println("Empate");
      } else if ((player.equals("Paper") && cpu.equals("Rock")) || (player.equals("Scissors") && cpu.equals("Paper"))
          || (player.equals("Rock") && cpu.equals("Scissors"))) {
        System.out.println("Player wins :)");
      } else {
        System.out.println("CPU wins :(");
      }

    } else {
      System.out.println("Option not available!");
    }

    scanner.close();
  }
}

File: Games/RockPaperScissors/RockPaperScissors.js
const readline = require("readline").createInterface({
	input: process.stdin,
	output: process.stdout
});

// Intro
console.log("Lets play a game:");
console.log(" ---- Rock Paper Scissors ----");
console.log("");

console.log("Paper -> Rock");
console.log("Scissors -> Paper");
console.log("Rock -> Scissors");
console.log("");

// Player selection
readline.question("You: ", (player) => {
	if (player === "Rock" || player === "Paper" || player === "Scissors") {
		// Randomize CPU
		const rdn = Math.floor(Math.random() * 3);
		let cpu = "";

		if (rdn === 0) cpu = "Rock";
		else if (rdn === 1) cpu = "Paper";
		else cpu = "Scissors";

		console.log("CPU: " + cpu);

		// Check Winner
		console.log("");
		if (player === cpu) console.log("Draw");
		else if (
			(player === "Paper" && cpu === "Rock")
			|| (player === "Scissors" && cpu === "Paper")
			|| (player === "Rock" && cpu === "Scissors")
		) console.log("Player wins :)");
		else console.log("CPU wins :(");

	} else console.log("Option not available!");

	readline.close();
});
